unlock-js/package.json

```diff
@@ -42,7 +44,10 @@
     "@babel/cli": "7.8.4",
     "@babel/core": "7.8.7",
     "@babel/preset-env": "7.8.7",
+    "@unlock-protocol/unlock-js": "0.8.7",
```

Comment:
I think we shold not require this dep here?

Reply:
I am actually not sure how "circular" dev dependencies work. Maybe that's fine?

Reply:
In the stand ups current form this is implicitly being used for its access to WalletService and latest.  

This is included as a development dependency to ensure that this change is a structural one to avoid the side effects of referencing the classes from the library itself now that it performs the test setup for itself. 

We can pull it but I think we should do it after this goes out to isolate any ripples